packages/grid/backend/grid/api/settings/settings.py

```diff
@@ -56,19 +57,19 @@ def create_setup(
     if isinstance(reply, ExceptionMessage):
         resp = {"error": reply.exception_msg}
     else:
-        resp = {"message": reply.resp_msg}
-
+        resp = {"message": reply.content}
     return resp
 
 
 @router.get("", status_code=200, response_class=JSONResponse)
-def get_setup(request: Request, current_user: Any = Depends(get_current_user)) -> Any:
+def get_setup(
+    current_user: UserPrivate = Depends(get_current_user),
```

Comment:
@IonesioJunior I'm unsure if this route requires authentication. Can you confirm?

Reply:
Fixed by 34b81b7ff93985f9b375ed4056dc426c4713d302